### PR TITLE
Add Zipkin tracing for http and messages

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: topic-gateway/src/test/java/io/sk8s/topic/gateway/TopicGatewayApplicationTests.java [hunk 1]
```diff
@@ -24,9 +24,11 @@
 
 /**
  * @author Mark Fisher
+ * @author Thomas Risberg
  */
-@RunWith(SpringRunner.class)
-@SpringBootTest
+// Need a JUnit Kafka Rule to avoid failing when Kafka not available
```
Comment (markfisher) [inline topic-gateway/src/test/java/io/sk8s/topic/gateway/TopicGatewayApplicationTests.java]: @trisberg did you create an issue for this? similar to https://github.com/markfisher/sk8s/issues/26

File: zipkin-server/src/main/java/io/sk8s/zipkin/server/demo/ZipkinServer.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package io.sk8s.zipkin.server.demo;
```
Comment (markfisher) [inline zipkin-server/src/main/java/io/sk8s/zipkin/server/demo/ZipkinServer.java]: needs copyright header

File: topic-gateway/src/test/java/io/sk8s/topic/gateway/TopicGatewayApplicationTests.java (hunk 1 above)
Comment (trisberg) [reply topic-gateway/src/test/java/io/sk8s/topic/gateway/TopicGatewayApplicationTests.java]: Have created one - #39

File: zipkin-server/src/main/java/io/sk8s/zipkin/server/demo/ZipkinServer.java (hunk 2 above)
Comment (trisberg) [reply zipkin-server/src/main/java/io/sk8s/zipkin/server/demo/ZipkinServer.java]: Added

**Comment by markfisher** [comment]
@trisberg can you please rebase this on the latest master?

**Comment by trisberg** [comment]
I tried but getting some really weird conflicts on files that weren't touched, strange :(

**Comment by trisberg** [comment]
Closed in favor of https://github.com/markfisher/sk8s/pull/45

**Pull request closed without merging**